cognite/client/_api/ai/tools/documents.py

```diff
@@ -0,0 +1,114 @@
+from __future__ import annotations
+
+from collections.abc import Sequence
+
+from cognite.client._api_client import APIClient
+from cognite.client.data_classes.ai import Answer, AnswerLanguage, Summary
+from cognite.client.utils._identifier import InstanceId
+from cognite.client.utils.useful_types import SequenceNotStr
+
+
+class AIDocumentsAPI(APIClient):
+    _RESOURCE_PATH = "/ai/tools/documents"
+
+    def summarize(
+        self,
+        ids: Sequence[int] | None = None,
+        external_ids: SequenceNotStr[str] | None = None,
+        instance_ids: Sequence[InstanceId] | None = None,
```

Comment:
I think files api use NodeId instead of instanceId

Reply:
nice catch. I think it might be a good idea to switch.

Reply:
done